src/frontend/http.rs

```diff
@@ -279,6 +439,57 @@ mod tests {
     const V2_ETAG: &str =
         "06d033ece6645de592db973644cf7357255f24536ff7b03c3b2ace10736f7636";
 
+    // Create a mock upload request and execute it
+    async fn mock_upload_request(
+        filename: &str,
+        mut file_content: Vec<u8>,
+        schema_name: &str,
+        table_name: &str,
+        schema_json: Option<&str>,
+        context: Arc<dyn SeafowlContext>,
+    ) -> Response<Bytes> {
+        let handler = filters(context);
+
+        let mut body: Vec<u8> = vec![];
+
+        if let Some(schema_json) = schema_json {
+            body = "--42\r\n\
```

Comment:
Is there something more standard here we can use for the multipart/form-data boundary (though it looks like the actual server code doesn't use a hardcoded boundary anyway)?

Reply:
In [theory](https://www.rfc-editor.org/rfc/rfc7578#section-4.1) there's no enforcement on the boundary other than it can't appear in the body. In practice I see it's usually something like `------------------------3d5af745b8be129c`